### PR TITLE
adds redpanda docker compose

### PR DESCRIPTION
**Pull request by oleksiikraievyi**

### CODE REVIEW

File: redpanda-docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: redpanda-quickstart
```
Comment (kant2002) [inline redpanda-docker-compose.yml]: Should I nitpick here ?

**Review by kant2002 — COMMENTED** [review]
Great!

File: redpanda-docker-compose.yml (hunk 1 above)
Comment (kant2002) [inline redpanda-docker-compose.yml]:
```suggestion
name: stereodb-redpanda-cluster
```

Comment (oleksiikraievyi) [reply redpanda-docker-compose.yml]: no p, commited))

**Pull request merged**